packages/lib-classifier/src/store/subjects/Subject/FreehandLineReductions/FreehandLineReductions.spec.js

```diff
@@ -0,0 +1,116 @@
+import { expect } from 'chai'
+import sinon from 'sinon'
+import { reducedEmptySubject, reducedSubject } from './mocks'
+
+import FreehandLineReductions from './FreehandLineReductions'
+
+xdescribe('Models > FreehandLineReductions', function () {
+
+  describe('with reductions', function () {
+    let reductionsModel
+
+    before(function () {
+      const response = {
+        workflow: {
+          subject_reductions: [{ data: reducedSubject }]
+        }
+      }
+      reductionsModel = FreehandLineReductions.create({
+        reducer: 'alice',
+        subjectId: '13971150',
+        workflowId: '5339',
+        reductions: [{ data: reducedSubject }]
+      })
+    })
+
+    it('should exist', function () {
+      expect(reductionsModel).to.be.ok()
+    })
+
+    it('should have transcribed lines', function () {
+      reductionsModel.reductions.forEach(reduction => expect(reduction.data.transcribed_lines).to.equal(10))
+      expect(reductionsModel.consensusLines(0)).not.to.be.empty()
+    })
+
+    it('should have points', function () {
+      reductionsModel.consensusLines(0).forEach(function (consensusLine) {
+        expect(consensusLine.points).to.be.a('array')
+        expect(consensusLine.points).not.to.be.empty
+      })
+    })
+
+    it('should have text options', function () {
+      reductionsModel.consensusLines(0).forEach(function (consensusLine) {
+        expect(consensusLine.textOptions).to.be.a('array')
+        expect(consensusLine.textOptions).not.to.be.empty
+      })
+    })
+
+    describe('transcribed lines', function () {
```

Comment:
'Transcribed lines' copied across again.

Reply:
Didn't fix tests as part of first POC commit. Fixed now.